packages/cli/src/lib/commands/auth.ts

```diff
@@ -0,0 +1,25 @@
+import { CommandDefinition } from "../../types";
+import { setConfigValue } from "../config";
+
+/**
+ * Initializes `auth` command.
+ *
+ * @param {Command} program Commander object.
+ */
+export const auth: CommandDefinition = (program) => {
+    const authCmd = program
+        .command("auth [command]");
+
+    /**
+    * Command `si host version`
+    * Get STH version number from package.json file
+    * Log: version number
+    */
+    authCmd
+        .command("set-token")
+        .argument("<token>")
+        .description("Set Platform auth token")
+        .action(async (token: string) => {
+            setConfigValue("token", token);
```

Comment:
shouldn't we add simple format validation for JWT before save to config?

Reply:
we can add simple API endpoint to platfrom to validate token or just check here if lenght is correct

Reply:
I would go for simple format check- lenght and number of dots in string